proactive-jupyter-kernel/pragma.py

```diff
@@ -356,10 +374,36 @@ def is_valid_get_result(data):
     raise ParameterError('Invalid parameters')
 
 
-def is_valid_submitted_jobs(data):
+def is_valid_list_submitted_jobs(data):
     pass
 
 
+def is_valid_show_resource_manager(data):
+    pattern_dimension = r"^\d+$"
+    pattern_path_cars = r"^[a-zA-Z0-9_\/\\:\.-]+$"
+    if ('width' in data and 'height' not in data) or ('width' not in data and 'height' in data):
+        raise ParsingError('Missing one of height/width parameters')
+    if 'width' in data and 'height' in data and \
+            not (re.match(pattern_dimension, data['width']) and re.match(pattern_dimension, data['height'])):
+        raise ParameterError('Invalid height/width parameters')
+    if 'host' in data and not re.match(pattern_path_cars, data['host']):
+        raise ParameterError('Invalid host parameter')
+    return
+
+
+def is_valid_show_scheduling_portal(data):
```

Comment:
Also if we can factorize `is_valid_show_resource_manager` and `is_valid_show_scheduling_portal`

Reply:
Same!